docs/OncoTree-Mapping-Tool.md

```diff
@@ -1,3 +1,7 @@
+# Contents
+* [OncoTree to OncoTree Mapping Tool](#oncotree-to-oncotree-mapping-tool)
+* [Ontology Mapping Tool](#ontology-mapping-tool)
```

Comment:
I suggest making the name a little more specific. To be parallel with the other tool we have, perhaps this could be called the "Ontology to Ontology Mapping tool"? Other options: "External Ontology Mapping Tool", "Cross Ontology Mapping Tool", "Inter-Ontology Mapping Tool"

Reply:
I like Cross ontology. Will change to that

Reply:
I will change it to Ontology to Ontology Mapping tool to keep it in sync with oncotree to oncotree mappint